jailer.go

```diff
@@ -364,7 +364,7 @@ func linkFileToRootFS(cfg *JailerConfig, dst, src string) error {
 
 // LinkFilesHandler creates a new link files handler that will link files to
 // the rootfs
-func LinkFilesHandler(rootfs, kernelImageFileName string) Handler {
+func LinkFilesHandler(rootfs, kernelImageFileName, initrdFilename string) Handler {
```

Comment:
Do we really need the `initrdFilename` parameter?

We could instead just use `filepath.Base` to get the filename from the path.

Reply:
removed that parameter, and reverted `NewNaiveChrootStrategy` changes below